internal/rbd/rbd_attach.go

```diff
@@ -235,7 +245,28 @@ func appendDeviceTypeAndOptions(cmdArgs []string, isNbd, isThick bool, userOptio
 	return cmdArgs
 }
 
-func createPath(ctx context.Context, volOpt *rbdVolume, cr *util.Credentials) (string, error) {
+// appendRbdNbdCliOptions append manditory options and convert list of useroptions
+// provided for rbd integrated cli to rbd-nbd cli format specific.
+func appendRbdNbdCliOptions(cmdArgs []string, userOptions string) []string {
+	if !strings.Contains(userOptions, "try-netlink") {
```

Comment:
`try-netlink`  and `reattach-timeout` are used in multiple places define a const for it?